### ISSUE TITLE
phar error: invalid path, contains star

### ISSUE THREAD

**Issue by MakG10** [picqer/php-barcode-generator#94]
I'm using Satis to host a private package registry, there seems to be an issue with packaging recent version into PHAR archive because of a filename with `*`:

```
Dumping package 'picqer/php-barcode-generator' in version 'dev-master'.
  - Installing picqer/php-barcode-generator (dev-master fd18d04): Cloning fd18d04d64 from cache

                                                                               
  [BadMethodCallException]                                                     
  Entry tests/verified-files/C128-1234567890abcABC-283*33.svg cannot be creat  
  ed: phar error: invalid path "tests/verified-files/C128-1234567890abcABC-28  
  3*33.svg" contains star                                                      
```

**Comment by Niksac** [comment]
The stars are causing issues in other places too.
On windows, composer is unable to install this because filenames cannot contain `*`

**Comment by casperbakker** [comment]
Sorry about that, will fix this in the next version.

**Comment by Niksac** [comment]
Thank you :)

**Comment by casperbakker** [comment]
Thanks for the patience. I just released [v2.0.1](https://github.com/picqer/php-barcode-generator/releases/tag/v2.0.1) with a bugfix.
